assets/javascripts/lib/apply-spoiler.js

```diff
@@ -25,19 +25,33 @@ function isInteractive(event) {
   return event.defaultPrevented || event.target.closest(INTERACTIVE_SELECTOR);
 }
 
+function no_text_selected() {
```

Comment:
```suggestion
function noTextSelected() {
```

Small thing, but could we make the method name `camelCase` so it follows the general convention used among `.js` files